### PR TITLE
Add important trash abilities

### PR DESCRIPTION
**Pull request by modulo11**
Adds several trash abilities as described in issue: https://github.com/BigWigsMods/LittleWigs/issues/15

### CODE REVIEW

File: Legion/TheArcway/Trash.lua [hunk 1]
```diff
@@ -1,50 +1,90 @@
...
+		226206, -- Arcane Reconstitution
+
+		--[[ Warp Shade ]]--
+		226206, -- Arcane Reconstitution
+		211115, -- Phase Breach
+
+		--[[ Forgotten Spirit ]]--
+		226269, -- Torment
+	}, {
+		[211217] = L.anomaly,
```
Comment (funkydude) [inline Legion/TheArcway/Trash.lua]: These duplicate headers aren't necessary.

**Comment by funkydude** [comment]
Some of those auto diffs are awful, but nothing really sticks out at me.

**Comment by modulo11** [comment]
After some testing I decided to remove the Forgotten Spirit/Torment message as this spell is chain-casted and the notification way too spammy.

File: Legion/TheArcway/Trash.lua [hunk 2]
```diff
@@ -1,50 +1,76 @@
...
+		--[[ Arcane Anomaly ]]--
+		211217, -- Arcane Slicer
+		226206, -- Arcane Reconstitution
+
+		--[[ Warp Shade ]]--
+		226206, -- Arcane Reconstitution
+		211115, -- Phase Breach
+	}, {
+		[211217] = L.anomaly,
+		[226206] = L.anomaly,
```
Comment (funkydude) [inline Legion/TheArcway/Trash.lua]: You've still got these doubled.

File: Legion/TheArcway/Trash.lua [hunk 3]
```diff
@@ -1,50 +1,74 @@
...
+	return {
+		--[[ Arcane Anomaly ]]--
+		211217, -- Arcane Slicer
+		226206, -- Arcane Reconstitution
+
+		--[[ Warp Shade ]]--
+		211115, -- Phase Breach
+	}, {
+		[211217] = L.anomaly,
+		[226206] = L.anomaly,
```
Comment (funkydude) [inline Legion/TheArcway/Trash.lua]: It's the headers you are duplicating, headers only need to be specified once, they will encompass everything until the next header is set.

Comment (modulo11) [inline Legion/TheArcway/Trash.lua]: ARG. Sorry, still learning. Should be OK now then I guess.

Comment (funkydude) [inline Legion/TheArcway/Trash.lua]: `226206, -- Arcane Reconstitution`
Re-adding that was a mistake. I suppose you are trying to show that both mobs use it? It's not that important, just list it under 1 mob and it's done.

Comment (modulo11) [inline Legion/TheArcway/Trash.lua]: Exactly. Both mobs can cast the same spell. I added it to Arcane Anomaly now.

**Pull request merged**